components/mpas-ocean/driver/ocn_comp_mct.F

```diff
@@ -3381,6 +3390,53 @@ subroutine datetime(cdate, ctime)!{{{
 
    end subroutine datetime!}}}
 
+
+   subroutine init_glc_z_levels(domain, err)
+      use glc_zocnclass_mod
+
+      implicit none
+      type (domain_type), pointer, intent(inout) :: domain
+      integer, intent(out) :: err
+
+      ! local vars
+      type (block_type), pointer :: block_ptr
+      type (mpas_pool_type), pointer :: forcingPool
+      integer, pointer :: nGlcZLevels
+      real (kind=RKIND), dimension(:), pointer :: glcZLevels
+      integer :: cpl_num_zocn, iLev
+
+      err = 0
+
+      block_ptr => domain % blocklist
+      do while(associated(block_ptr))
+         call mpas_pool_get_subpool(block_ptr % structs, 'forcing', forcingPool)
+         call mpas_pool_get_dimension(forcingPool, 'nGlcZLevels', nGlcZLevels)
+         if (nGlcZLevels > 0) then
+            call mpas_pool_get_array(forcingPool, 'glcZLevels', glcZLevels)
+
+            cpl_num_zocn = glc_get_num_zocn_classes()
+
+            ! check that the num z-levels in the coupler matches what MALI was told to use
+            if (nGlcZLevels /= cpl_num_zocn) then
+               call mpas_log_write("nGlcZLevels=$i does not match glc_get_num_zocn_classes=$i", &
+                                   MPAS_LOG_ERR, intArgs=(/nGlcZLevels, cpl_num_zocn/))
+               err = ior(err, 1)
+            endif
+
+            glcZLevels = -1.0_RKIND * glc_get_zlevels()
```

Comment:
As discussed in https://github.com/E3SM-Ocean-Discussion/E3SM/pull/121#discussion_r1997908583, let's not do this sign flip.